_episodes/10-actel-intro.md

```diff
@@ -114,7 +114,7 @@ OK, now we have a biometrics file of our tag releases with names for each place
 setwd('actel_example')
 
 # Run analysis. Note: This will open an analysis report on your web browser.
-exp.results <- explore(tz = 'US/Central', report = TRUE)
+exp.results <- explore(tz = 'Europe/Copenhagen', report = TRUE)
```

Comment:
thats the timezone we need?

Reply:
this example dataset uses Hugos data, and this is what gh-pages has

Reply:
sounds good